edit-post/store/selectors.js

```diff
@@ -16,24 +21,42 @@ export function getEditorMode( state ) {
 }
 
 /**
- * Returns the current active panel for the sidebar.
+ * Returns true if the editor sidebar is opened.
  *
- * @param {Object} state Global application state.
+ * @param {Object} state Global application state
+ * @return {boolean}     Whether the editor sidebar is opened.
+ */
+export function isEditorSidebarOpened( state ) {
+	const activeGeneralSidebar = getPreference( state, 'activeGeneralSidebar', null );
+
+	return [ 'edit-post/document', 'edit-post/block' ].includes( activeGeneralSidebar );
```

Comment:
To check if a sidebar is active ( the general one) we check a property of the sidebar (which of tabs is opened). That seems like a good improvement to do next. The logic to check if the general sidebar is oppened or not should not rely on an internal property of the general sidebar.
We can have one common logic to check which sidebar should be rendered common to all sidebars.
The tab of the general sidebar is only used when rendering that sidebar, it just a config of that sidebar. The tab logic of the general sidebar should be specified in an identical way to what a plugin needing tabs would do.

Reply:
`Array#includes` is not polyfilled. This will error in IE11. Use [`_.includes`](https://lodash.com/docs/4.17.4#includes) instead.

Reply:
Just wanted to clarify as I'm about to sign off for the weekend and apparently not thinking clear enough :)

Do you propose to combine those two sidebars into one that contains two tabs and display only one depending on the internal state (block vs document)?

Reply:
I will add a commit in the PR https://github.com/WordPress/gutenberg/pull/5528 that also solves this includes problem.

Reply:
Thanks for fixing ❤️